robottelo/ui/factory.py

```diff
@@ -124,11 +124,11 @@ def make_org(session, force_context=False, **kwargs):
         u'select': True,
     }
     page = session.nav.go_to_org
-    core_factory(create_args, kwargs, session, page, force_context=False)
+    core_factory(create_args, kwargs, session, page)
     Org(session.browser).create(**create_args)
 
 
-def make_loc(session, force_context=False, **kwargs):
+def make_loc(session, force_context=True, **kwargs):
```

Comment:
> *) Updated ui/factory force_context as Admin User is now placed
> in Default_Organization org by default..

In some places, `force_context` now defaults to true. In other places, `force_context` has been dropped entirely. I don't really understand what's going on. Can you explain a bit further?